contrib/docker/Dockerfile

```diff
@@ -0,0 +1,82 @@
+
+# TODO: Needs some documentation here about the RPC server, logging via docker
+# logs, mounting a volume, the conf file, etc...
+
+###############
+# Builder Stage
+###############
+
+# Basic Go environment with git, SSL CA certs, and upx.
+# golang:1.17.0-alpine
+FROM golang@sha256:c03a922dfb0d6d3a94d46cc5be69002011763595ac2786068fde63ee174d797b AS builder
+RUN apk add --no-cache git ca-certificates upx
+
+# Empty directory to be copied into place in the production image since it will
+# run as a non-root container and thus not have permissions to create
+# directories or change ownership of anything outside of the structure already
+# created for it.
+RUN mkdir /emptydatadir
+
+# New unprivileged user for use in production image below to improve security.
+ENV USER=decred
+ENV UID=10000
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home="/home/${USER}" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    "${USER}"
+
+# Build dcrd and other commands it provides
+WORKDIR /go/src/github.com/decred/dcrd
+RUN git clone https://github.com/decred/dcrd . && \
+    CGO_ENABLED=0 GOOS=linux \
+    go install -trimpath -tags safe,netgo -ldflags="-s -w" \
```

Comment:
since it doesn't appear that timezone data is being copied from the builder, i'd suggest building everything (or at the very least dcrd) with the `timetzdata` build tag.

Reply:
I went ahead and updated it to add the tag.  I don't think it matters though since I checked and none of the code actually uses it which is why I didn't add the tzdata.